src/synonyms.ts

```diff
@@ -0,0 +1,61 @@
+import * as _request from 'request-promise';
+import * as fs from 'fs';
+import * as Ajv from 'ajv';
+import { request } from 'https';
+
+
+export default async function synonyms({
+   payload }: { payload: { symbol: string } }) {
+
+  const data: any = payload;
+  const symbol: string = payload.symbol;
+  const url_symbol = `https://rest.genenames.org/fetch/symbol/${symbol}`;
+
+  const options_symbol = {
+    method: 'GET',
+    uri: url_symbol as string,
+    json: true
+  };
+
+  const result: any = await _request(options_symbol)
+    .then((res: any) => {
+      if (res.response.docs.length === 1 && typeof res.response.docs[0] !== 'undefined') {
+          return res.response.docs[0].alias_symbol.concat(res.response.docs[0].symbol);
+      }else {
+        const json_alias = retrieveAliasSynonyms({ symbol_alias : symbol });
+        return json_alias;
```

Comment:
You can just return the line above, you don't need to assign and then return it.

Reply:
Done.